### DISCUSSION TITLE
Adding a new class .children:flex-1 does not work but it should?

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/tailwindcss#7197] [Help]
I wanted to add a new tailwind utility class `children:flex-1` but it is not working (not being added or applied). Is this a bug or some kind of internal conflict in Tailwind or am I doing it wrong? If I rename it to `children-flex-1` then it works but I wanted the class to be called `children:flex-1` to be more consistent with Tailwind's syntax.

**tailwind.css**

````
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer utilities {
    .children\:flex-1 > * {
        flex: 1;
    }
}
````

**html**

````
<div class="flex children:flex-1">
  <div>one</div>
  <div>two</div>
</div>
````

This _should_ apply `flex: 1;` to both divs inside of the container but it's not working.

**Comment by macbruker** [comment]
Not sure why it doesn't work inside a utilities layer. Until that part is solved, I suggest dropping the layer wrapper and just write it like this.
```css
.children\:flex-1 > * {
  flex: 1;
}
```
